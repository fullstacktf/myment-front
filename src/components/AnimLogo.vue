<template>
  <div class="logo"></div>
</template>

<script>
export default {
  name: 'AnimLogo',
};
</script>

<style>
.logo {
  justify-self: center;
  align-self: center;
  width: 700px;
  height: 700px;
  border: 4px solid grey;
}
</style>
