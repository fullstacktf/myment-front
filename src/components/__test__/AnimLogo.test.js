import AnimLogo from '../AnimLogo.vue';
import { shallowMount, createLocalVue } from '@vue/test-utils';
import Buefy from 'buefy';
import mock from '../../test/mockBack';

describe('AnimLogo', () => {
  let localVue;
  beforeEach(() => {
    const Vue = createLocalVue();
    Vue.use(Buefy);
    localVue = Vue;
  });
  test('render component', () => {
    const wrapper = shallowMount(AnimLogo, {
      localVue,
    });
    expect(wrapper.html()).toMatchSnapshot();
  });

  test('Check props', () => {});

  test('check Contents props', () => {});
});
