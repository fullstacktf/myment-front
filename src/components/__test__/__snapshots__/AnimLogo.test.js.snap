// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnimLogo render component 1`] = `"<div class=\\"logo\\"></div>"`;
